transpose.json

```diff
@@ -0,0 +1,186 @@
+{
+  "#": [
+      "JSON doesn't allow for multi-line strings, so all multi-line input is ",
+      "presented here as arrays of strings. It's up to the test generator to join the ", 
+      "lines together with line breaks."
+  ],
```

Comment:
Perhaps it should be: `JSON doesn't allow for elegant formatting of multi-line strings, so all multi-line input and output is`...